features/README.md

```diff
@@ -7,10 +7,12 @@
 
-| Example | Programming Languages | Description |
-|:---------|:---------|:---------|
-| [Access Controls](share/access-controls/README.md) | JavaScript, GraphGQL| Explore each access control mechanism used by Vendia Share through the series of examples |
-| [GraphQL](share/graphql/README.md) | Golang, Python, JavaScript, GraphQL | Explore basic querying and changing data in a Uni using several programming languages | 
-| [GraphQL Enhancements](share/graphql-enhancements/README.md) | Golang, Python, JavaScript, GraphQL | Explore GraphQL enhancements (like conditional puts and transactions) to conventional GraphQL functionality using several programming languages | 
-| [GraphQL Filters](share/graphql-filters/README.md) | GraphQL | Explore the GraphQL filter syntax in depth | 
-| [Smart Contracts](share/smart-contracts/README.md) | JavaScript, GraphQL | Explore a set of smart contracts that are used for data validation, computation, and enrichment |
+| Example                                                       | Programming Languages               | Description                                                                                                                                     |
+|:---------                                                     |:---------                           |:---------                                                                                                                                       |
+| [Access Controls](share/access-controls/README.md)            | JavaScript, GraphGQL                | Explore each access control mechanism used by Vendia Share through the series of examples                                                       |
+| [GraphQL](share/graphql/README.md)                            | Golang, Python, JavaScript, GraphQL | Explore basic querying and changing data in a Uni using several programming languages                                                           | 
+| [GraphQL Enhancements](share/graphql-enhancements/README.md)  | Golang, Python, JavaScript, GraphQL | Explore GraphQL enhancements (like conditional puts and transactions) to conventional GraphQL functionality using several programming languages | 
+| [GraphQL Filters](share/graphql-filters/README.md)            | GraphQL                             | Explore the GraphQL filter syntax in depth                                                                                                      | 
+| [Smart Contracts](share/smart-contracts/README.md)            | JavaScript, GraphQL                 | Explore a set of smart contracts that are used for data validation, computation, and enrichment                                                 |
+| [Notifications](share/notifications/README.md)                | GraphQL                             | Explore ways to receieve notifications from Vendia through both UI and graphQL                                                                  |
```

Comment:
Also...is it _really_ receiving notifications through GraphQL?

Reply:
Nope. That's just my bad English. Let me correct that.